<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Second_windows</class>
 <widget class="QDialog" name="Second_windows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>533</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="main_title_lable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>193</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Creare indicator</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="name_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Crearea indicatorului</string>
   </property>
   <widget class="QPushButton" name="save_button_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvare!</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numele proiectului</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="title_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="link_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Unde salvați indicatorul?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="link_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="add_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>481</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Adăugarea indicatorilor de performanță</string>
   </property>
   <widget class="QLabel" name="time_set_lable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data inregistrării:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="time_set_data">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="progress_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cât ați făcut? (unități)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="end_button_2">
    <property name="geometry">
     <rect>
      <x>374</x>
      <y>160</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Adăugare element</string>
    </property>
   </widget>
   <widget class="QLabel" name="proposed_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cât v-ați propus să faceți? (unități)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="progress_spin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="proposed_spin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>end_button_2</zorder>
   <zorder>time_set_lable</zorder>
   <zorder>time_set_data</zorder>
   <zorder>progress_label</zorder>
   <zorder>proposed_label</zorder>
   <zorder>progress_spin</zorder>
   <zorder>proposed_spin</zorder>
  </widget>
  <widget class="QGroupBox" name="time_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>481</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Timpul de lucru</string>
   </property>
   <widget class="QRadioButton" name="time_unit_month">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Luni</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvare</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="time_unit_spin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="time_unit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cât timp aveți la dispoziție?</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_unit_lable_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ce unitate de măsură folosiți?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="time_unit_year">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ani</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
